### ISSUE TITLE
Typo in condition

### ISSUE THREAD

**Issue by SvyatoslavRazmyslov** [moq/moq4#236]
Due to an typo, there is no comparison with y.Method.

[V3001](http://www.viva64.com/en/d/0381/) There are identical sub-expressions 'x.Method' to the left and to the right of the '==' operator. ExpressionComparer.cs 165
```C#
private bool EqualsBinary(BinaryExpression x, BinaryExpression y)
{
  return x.Method == x.Method && this.Equals(x.Left, y.Left) && this.Equals(x.Right, y.Right) && this.Equals(x.Conversion, y.Conversion);
}
```
This place is found by a static analyzer [PVS-Studio](http://www.viva64.com/en/pvs-studio/).

**Comment by LeonidLevin** [comment]
I believe this has been solved in commit 13a5362a1c4ca0508e419fbfaf44df7b738a6235
Please, mark as closed.

**Comment by kzu** [comment]
Thanks!
